static/src/js/actions/accessMethods.js

```diff
@@ -49,6 +50,12 @@ export const fetchAccessMethods = () => (dispatch, getState) => {
             selectedAccessMethod
           }))
         })
+        .catch((error) => {
+          dispatch(actions.handleError(error, 'access methods'))
+
+          console.error('Failed to fetch access methods', error)
```

Comment:
Sometimes we claim the promise was rejected (it wasn't, that would imply an http error), other actions simply `console.error(error)` so lets just be consistent in these, I'd suggest an informative prefix [`console.error('Action [ACTION_NAME] failed:', error)`] (The specific action I think is helpful here, versus a message about the action its taking)